tests/test_utils/tests.py

```diff
@@ -19,7 +19,7 @@
 )
 from django.test.html import HTMLParseError, parse_html
 from django.test.utils import (
-    CaptureQueriesContext, isolate_apps, override_settings,
+    CaptureQueriesContext, TestContextDecorator, isolate_apps, override_settings,
```

Comment:
isort is failing, so just move `TestContextDecorator` to the end of the line here.